packages/core/src/utils/util.ts

```diff
@@ -25,3 +25,9 @@ export const enumToObjectByValue = (enumType: EnumInterface) => {
     .map(({ name, value }) => ({ [value]: name }))
     .reduce((acum, entry) => ({ ...acum, ...entry }), {})
 }
+
+export const inputId = (): string =>
```

Comment:
change this name to generateId or similar, since this Id works for other stuff too